src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Transport.php

```diff
@@ -101,7 +101,7 @@ protected function doSend(MessageInterface $message): SentMessage
                 'Content-Type' => 'application/json',
                 'User-Agent' => 'LOX24 Symfony Notifier',
             ],
-            'body' => $body,
+            'body' => json_encode($body, JSON_THROW_ON_ERROR),
```

Comment:
```suggestion
            'json' => $body,
```

Should also work, right?